user-interface/src/main/java/life/qbic/datamanager/views/projects/create/ExperimentalInformationLayout.java

```diff
@@ -61,11 +81,16 @@ public ExperimentalInformationLayout(
         .bind(ExperimentalInformation::getExperimentName,
             ExperimentalInformation::setExperimentName);
 
+    Div speciesRow = new Div(speciesIconBox, speciesBox);
+    speciesRow.addClassName("input-with-icon");
+    Div specimenRow = new Div(specimenIconBox, specimenBox);
+    specimenRow.addClassName("input-with-icon");
```

Comment:
As far as I can tell `input-with-icon` is not needed

Reply:
it is, but I renamed it to better reflect what this style is needed for